README.md

```diff
@@ -101,8 +101,11 @@ On CoreOS Container Linux, the supported providers and metadata are [somewhat di
       - AFTERBURN_PACKET_HOSTNAME
       - AFTERBURN_PACKET_PLAN
       - AFTERBURN_PACKET_IPV4_PUBLIC_0
+      - AFTERBURN_PACKET_IPV4_PUBLIC_GW_0
```

Comment:
Here and everywhere, I'd probably spell out `GATEWAY` entirely.

Reply:
Done.